tests/util.go

```diff
@@ -82,6 +82,25 @@ const DefaultServerConfig = `
 </hazelcast>
 `
 
+const LiteMemberConfig = `
```

Comment:
I do not like the idea of having xml configs inlined like this. I think required xml configs should be in their respective `.xml` file. One reason being that you lose syntax highlighting in xml which makes it harder to read. Also, these are not really `util` kind of things that are some fundamental helpers used by multiple tests. Rather, each one is expected to be used by a single test most of the time.

On the other hand, this way it is easier to find which tests use a specific configuration.

I would say put here a helper function that will read the content of a given xml file into a string. Keep actual xml files close to the tests that use them.